pkg/build/build.go

```diff
@@ -0,0 +1,86 @@
+package build
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func Build(imageConfig *config.ImageConfig, buildConfig *config.BuildConfig) error {
+	err := prepareBuildDir(buildConfig)
+	if err != nil {
+		return err
+	}
+
+	err = ConfigureMessage(buildConfig)
+	if err != nil {
+		return err
+	}
+
+	err = cleanUpBuildDir(buildConfig)
+	return err
+}
+
+//go:embed scripts/script_base.sh
+var scriptBase string
+
+
+func prepareBuildDir(buildConfig *config.BuildConfig) error {
+
+	/* Combustion works by creating a volume with a subdirectory named "combustion"
+	   and a file named "script". This function builds out that structure and updates
+	   the BuildConfig so that the other functions can populate it as necessary.
+	 */
```

Comment:
Nothing major, there's an extra space in front of the `*/`

Reply:
Fixed